tests/test_missing.py

```diff
@@ -6,63 +6,317 @@
 import tempfile
 import unittest
 
+from netCDF4 import Dataset
+
 from activestorage.active import Active
 from activestorage.config import *
 from activestorage import dummy_data as dd
 
 import utils
 
 
-def _doit(testfile):
-    """ 
-    Compare and contrast vanilla mean with actual means
-    """
-    uri = utils.write_to_storage(testfile)
-    active = Active(uri, "data", utils.get_storage_type())
+"""
+There is a high level of duplication in this test module:
+this is to avert any issues of I/O toestepping from h5netcdf
+when running with USE_S3=True; this will probably have to change
+when we start using netCDF4python with S3-enabled
```

Comment:
It's a shame to add the duplication. How do you think it resolves IO issues?

Reply:
I am just about to de-duplicate, I am 99.99% sure the issues we were seeing were due to not closing instances of `Dataset` and I think I've fixed those all (I wrote that doc before I identified all those hanging open Datasets) :+1: